data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/ExecuteVaultNamedQueryRequest.avsc

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "record",
+  "name": "ExecuteVaultNamedQueryRequest",
+  "doc": "Retrieve and execute a named query with the given parameters.",
+  "namespace": "net.corda.data.ledger.persistence",
+  "fields": [
```

Comment:
Do we need this? Can `FindWithNamedQuery` be used instead?

Reply:
hmm true, I think it should work